service/frontend/nexus_handler.go

```diff
@@ -517,6 +553,7 @@ func (h *nexusHandler) CancelOperation(ctx context.Context, service, operation,
 	if err != nil {
 		return err
 	}
+	oc.enrichNexusOperationMetrics(service, operation, options.Header)
```

Comment:
Looks like we're missing a call to `augmentContext` here, could you please add that while you're here?